### PR TITLE
OWTimeSlice: Svg icons -> Font

### PR DESCRIPTION
**Pull request by irgolic**
The cherry on top of #118.

This is done to get the correct icon coloring on the buttons.

There's no way of telling how an OS colors the button, bar checking which operating system it is. This is a problem, because macOS will sometimes color the background blue, and sometimes white, making both black icons and white icons inappropriate. Text is automatically painted in the appropriate color, so this is the most elegant solution.

### CODE REVIEW

**Comment by ajdapretnar** [comment]
This doesn't work for me (same error as in Travis). Please check.

**Comment by irgolic** [comment]
Sorry, forgot to commit the font.

File: orangecontrib/timeseries/widgets/owtimeslice.py [hunk 1]
```diff
@@ -1,14 +1,13 @@
...
 
-import operator
 from collections import OrderedDict
 from numbers import Number
 from os.path import join, dirname
 
 from AnyQt.QtWidgets import QLabel, QDateTimeEdit
-from AnyQt.QtCore import QDateTime, Qt, QSize, QTimer, QTimeZone
-from AnyQt.QtGui import QIcon
+from AnyQt.QtCore import QDateTime, Qt, QSize, QTimer, QTimezone
```
Comment (ajdapretnar) [inline orangecontrib/timeseries/widgets/owtimeslice.py:9]: This is indeed QTimeZone, not QTimezone.

**Comment by ajdapretnar** [comment]
I have merged this as is, but there is now a slight change of button size when changing from pause to play (and vice versa).

**Pull request merged**